rules-tests/CodingStyle/Rector/Catch_/CatchExceptionNameMatchingTypeRector/Fixture/change_variable_next_try_reassign_same.php.inc

```diff
@@ -1,35 +0,0 @@
-<?php
-
-namespace Rector\Tests\CodingStyle\Rector\Catch_\CatchExceptionNameMatchingTypeRector\Fixture;
-
-class ChangeVariableNextTryReassignSameVariable
-{
-    public function run()
-    {
-        try {
-        } catch (SomeException $typoException) {
-        }
-
-        $typoException = $typoException;
-    }
-}
-
-?>
------
-<?php
-
-namespace Rector\Tests\CodingStyle\Rector\Catch_\CatchExceptionNameMatchingTypeRector\Fixture;
-
-class ChangeVariableNextTryReassignSameVariable
-{
-    public function run()
-    {
-        try {
-        } catch (SomeException $someException) {
-        }
-
-        $someException = $someException;
```

Comment:
This is extreemely rare case, which should be handled by another dead-code rule. Not neccesary to cover it

Reply:
I have use case for this, by disable this case, it will cause error on next usage, see

https://github.com/samsonasik/ErrorHeroModule/blob/a7e7e3fe2205e498ad2c689d802382cec48476d4/src/Middleware/Mezzio.php#L55-L62


```php
        try {
            $this->request = $serverRequest;
            $this->phpError();
            return $requestHandler->handle($serverRequest);
        } catch (Throwable $throwable) {
        }

        return $this->exceptionError($throwable);
```

Reply:
Oh, I see, it is about re-assign, it is ok then 👍